core/genesis.go

```diff
@@ -416,12 +416,18 @@ func (g *Genesis) ToBlock() *types.Block {
 	if err != nil {
 		panic(err)
 	}
+
+	extraData := g.ExtraData
+	if g.Config.AstriaOverrideGenesisExtraData {
```

Comment:
should this be in the genesis file (ie. `genesis.json` which is used as an example genesis?)

Reply:
Done.